src/ui-lib/sass/05-components/03-organisms/00-page-structure/_01-page-footer.scss

```diff
@@ -26,21 +25,23 @@ body > footer {
   }
 
   svg {
-    display: block;
+    display: inline-block;
+    margin: 0 auto;
     width: auto;
     height: 1em;
-    margin: 0 auto;
   }
 
   .grav-c-logo {
     border-bottom: 1px solid $grav-co-neutral-dirty-snow-white;
+    max-width: 7rem;
 
     &:hover {
       border-bottom-color: $grav-co-neutral-ashtray;
     }
   }
 
   .grav-c-icon {
+    max-width: 1.5em;
```

Comment:
As per my big comment - you _should_ be able to replace setting `max-width` and `height` here with simply setting `width: 1.5em;`.